.devops/pagopa-deploy-pipelines.yml

```diff
@@ -53,7 +53,7 @@ resources:
     - repository: checkoutTests
       type: github
       name: pagopa/pagopa-checkout-tests
-      ref: main
+      ref: CHK-3353-fix-integration-tests
```

Comment:
Revert to main after [checkout-tests PR](https://github.com/pagopa/pagopa-checkout-tests/pull/103) merge

Reply:
done here f722b5fa0475bc6e022c7f0e6a227024268beedc